corehq/apps/reports/filters/users.py

```diff
@@ -360,6 +360,7 @@ def user_es_query(cls, domain, mobile_user_and_group_slugs, request_user):
             user_type_filters.append(user_es.admin_users())
         if HQUserType.UNKNOWN in user_types:
             user_type_filters.append(user_es.unknown_users())
+            user_type_filters.append(user_es.system_users())
```

Comment:
I wonder if we want to expose this as a separate option too, it seems like most people know they are looking for `system` when looking for HQ submitted forms

Reply:
That'd be good to raise with @djmore9 - he's been looking at this filter a bit lately